docs/nos-client/api.md

```diff
@@ -228,4 +207,79 @@ nos.send({ asset: GAS, amount, receiver })
     .then((txid) => alert(`${amount} GAS sent in transaction ${txid}`))
     .catch((err) => alert(`Error: ${err.message}`));
 ```
+
+
+## `getPublicKey`
 ---
+The `getPublicKey` provides the public key of the currently authenticated account. It does not require the user to grant permission.
+
+#### Parameters
+None.
+
+#### Returns
+**string** - The public key of the currently signed in user.
+
+#### Example
+```javascript
+const nos = window.NOS.V1;
+
+nos.getPublicKey()
+  .then((publicKey) => alert(`Public Key: ${publicKey}`))
+  .catch((err) => alert(`Error: ${err.message}`));
+```
+
+
+## `encrypt`
+---
+The `encrypt` function allows you to encrypt arbitrary data for another user (you will need his public key, please see `getPublicKey`). It does not require the user to grant permission.
```

Comment:
```
you will need his public key, please see `getPublicKey`
```
It returns your own pubkey right, It's a little confusing with the text written here